bin/n

```diff
@@ -76,6 +76,9 @@ N_NODE_MIRROR=${N_NODE_MIRROR:-${NODE_MIRROR:-https://nodejs.org/dist}}
 N_NODE_MIRROR=${N_NODE_MIRROR%/}
 readonly N_NODE_MIRROR
 
+N_ARCH="${N_ARCH-}"
+readonly N_ARCH
+
```

Comment:
Perhaps leave this out? In the other cases like `N_NODE_MIRROR`, the global variable is set to a default value if not supplied and/or normalised. Nothing extra to do with `N_ARCH`.

There are other globals (from env) that are just used in the code like `N_PRESERVE_COREPACK`.

(I have wondered about having all the globals mentioned here, but have not established that as a pattern.)

Reply:
OK.  TBH, I couldn't figure out if you'd prefer a consistent declaration thing or not.  (And it actually had an effect on the code: I wanted to set it due to the version thing, but didn't do that to keep it readonly.)

In any case, I removed it.